test-with-jest/sample.env

```
@@ -0,0 +1,9 @@
+SUBSCRIPTION_ID=<your-subscription-ID>
+RESOURCE_GROUP_NAME="my-resource-group
+LOCATION=eastus2
+COSMOS_DATABASE_NAME="my-db"
+COSMOS_CONTAINER_NAME="my-container"
```

```suggestion
SUBSCRIPTION_ID=<your-subscription-ID>
RESOURCE_GROUP_NAME=<your-resource-group>
LOCATION=<resource-group-location>
COSMOS_DATABASE_NAME=<your-database-name>
COSMOS_CONTAINER_NAME=<your-cosmos-container-name>
```

or something along those lines?